src/types/object.ts

```diff
@@ -1 +0,0 @@
-export type ValueOf<T> = T[keyof T]
```

Comment:
I'd probably keep this one as it can come quite handy and we might want to add other object generic type helpers